app/models/hackathon/digest/listings/criterion/location.rb

```diff
@@ -22,7 +22,7 @@ def hackathons_in(location)
       end
     end
 
-    def hackathons_near(location)
+    def hackathons_near(location, coordinates)
```

Comment:
We can pass in `subscription`, then pull out the location and coordinates using `subscription.to_location` and `subscription.coordinates`

Reply:
I thought about that and didn't see an existing coordinates method, but I'm doing some refactoring in a bit and will definitely clean that up.